windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -717,17 +754,16 @@ procedure TMergeKeyboardInfo.CheckOrAddMinKeymanVersion;
 begin
   MinVersion := $0500;
   // For each .kmx, get minimum version and add to the result
-  for i := 0 to High(FKMXFiles) do
-    if FKMXFiles[i].Info.FileVersion > MinVersion then
-      MinVersion := FKMXFiles[i].Info.FileVersion;
+  for i := 0 to High(FPackageKMXFileInfos) do
+    if FPackageKMXFileInfos[i].Info.FileVersion > MinVersion then
+      MinVersion := FPackageKMXFileInfos[i].Info.FileVersion;
 
   FJSMinVersionString := '';
-  for i := Low(FJSFiles) to High(FJSFiles) do
   begin
-    with TRegEx.Match(FJsFiles[i].Data, 'this\.KMINVER=([''"])([^''"]+)(\1)') do
+    with TRegEx.Match(FJSFileInfo.Data, 'this\.KMINVER=([''"])([^''"]+)(\1)') do
```

Comment:
Indentation?

We may also need to run this check on the .js keyboards inside the package I think.